eq-author-api/tests/utils/contextBuilder/questionnaire/toggleQuestionnaireStarred.js

```diff
@@ -0,0 +1,19 @@
+const executeQuery = require("../../executeQuery");
+
+const mutation = `
+mutation ToggleQuestionnaireStarred($input: ToggleQuestionnaireStarredInput!) {
```

Comment:
that's a mouthful

Reply:
A mouthful of a thousand characters begins with a single byte